lib/travis/testing/factories.rb

```diff
@@ -65,7 +65,7 @@
   factory :enginex, :parent => :repository do
     name 'enginex'
     owner_name 'josevalim'
-    owner_email 'josevalim@email.com'
+    owner_email 'josevalim@email.example.com'
```

Comment:
It's bad form to use random public domains that aren't controlled/associated w/ the actual email address. It can easily result in spamming some poor domain.
`example.com` is one of a number of domains reserved for this purpose.